### ISSUE TITLE
About the loss compute

### ISSUE THREAD

**Issue by Howardqlz** [CuriousAI/mean-teacher#39]
Hi, i am wonder about this loss:
` class_loss = class_criterion(class_logit, target_var) / minibatch_size `
since this loss ignore some samples(no_lable), why here still use the minibatch_size not the labeled_size?
